release/prepareRelease.sh.cmake

```diff
@@ -195,7 +198,7 @@ case "$SYSTEM_NAME" in
 		for LIBSUFFIX in miniupnpc.5 boost_system boost_filesystem boost_iostreams boost_thread boost_locale boost_program_options ; do
```

Comment:
here `I` called the variable wrong (its LIBNAME or something like that, its not a suffix ;-) perhaps you could change that for me. thx